CH32V103/FreeRTOS/README_pio.md

```diff
@@ -1,7 +1,7 @@
 ---
 sys: reertos
```

Comment:
Wait what?
```suggestion
sys: freertos
```
Also, _zh?

Reply:
This is an existing problem. Sorry I didn't notice it.